views/project/invoice.erb

```diff
@@ -1,88 +1,55 @@
+        <dt class="text-right font-semibold text-gray-800 col-span-2">#{name}:</dt>
+        <dd class="text-gray-500" id="invoice-#{name.downcase.tr(" ", "-")}">#{value}</dd>
+      </dl>
+    CONTENT
+  end.join
+%>
 
-        <div class="mt-8 flex justify-end">
-            <div class="grid gap-2 text-right">
-              <dl class="grid grid-cols-3 gap-x-3">
-                <dt class="text-right font-semibold text-gray-800 col-span-2">Subtotal:</dt>
-                <dd class="text-gray-500"><%= @invoice_data[:subtotal] %></dd>
-              </dl>
-              <% if @invoice_data[:discount] != "$0.00" %>
-                <dl class="grid grid-cols-3 gap-x-3">
-                  <dt class="text-right font-semibold text-gray-800 col-span-2">Discount:</dt>
-                  <dd class="text-gray-500" id="invoice-discount">-<%= @invoice_data[:discount] %></dd>
-                </dl>
-              <% end %>
-              <% if @invoice_data[:credit] != "$0.00" %>
-                <dl class="grid grid-cols-3 gap-x-3">
-                  <dt class="text-right font-semibold text-gray-800 col-span-2">Credit:</dt>
-                  <dd class="text-gray-500" id="invoice-credit">-<%= @invoice_data[:credit] %></dd>
-                </dl>
-              <% end %>
-              <% if @invoice_data[:free_inference_tokens_credit] != "$0.00" %>
-                <dl class="grid grid-cols-3 gap-x-3">
-                  <dt class="text-right font-semibold text-gray-800 col-span-2">Free Inference Tokens:</dt>
-                  <dd class="text-gray-500" id="invoice-free-inference-tokens">-<%= @invoice_data[:free_inference_tokens_credit] %></dd>
-                </dl>
-              <% end %>
-              <dl class="grid grid-cols-3 gap-x-3 text-2xl">
-                <dt class="text-right font-semibold text-gray-800 col-span-2">Total:</dt>
-                <dd class="text-gray-500"><%= @invoice_data[:total] %></dd>
-              </dl>
-            </div>
-        </div>
-      </div>
-    </div>
+<div class="grid gap-6">
+  <div class="md:flex md:items-center md:justify-between text-2xl">
+    <div class="min-w-0 flex-1">This invoice will be finalized on the first day of next month</div>
+    <div class="text-right"><%= @invoice_data[:begin_time] %> to <%= @invoice_data[:end_time] %></div>
   </div>
+
+  <!-- Invoice Card -->
+  <%== part(
+      "components/table_card",
+      headers: ["Resource", "Description", "Usage", "Amount"],
+      rows: @invoice_data[:items].map do |item|
+         [
+           [
+             item[:name],
+             item[:description],
+             [item[:usage], { extra_class: "text-right"}],
+             [item[:cost_humanized], { extra_class: "text-right"}],
+           ],
+           { id: "item-#{item[:name]}" }
+         ]
+      end,
+      empty_state: "No resources",
+      footer: '<div class="px-4 py-5 sm:p-6 flex justify-end"><div class="grid gap-2 text-right">' + footer_content + "</div></div>"
```

Comment:
I recommend a double-quoted string and `#{footer_content}` interpolation here (not a big deal, just a bit more efficient).